CRM/Volunteer/Form/Defaults.php

```diff
@@ -18,32 +18,16 @@ function buildQuickForm() {
       $profileList[$profile['id']] = $profile['title'];
     }
 
-    $this->add(
-      'select',
-      'volunteer_default_profiles_individual',
-      ts('Default Profile(s) for Individual Registration'),
-      $profileList,
-      false, // is required,
-      array("placeholder" => ts("-- No Default profiles --"), "multiple" => "multiple", "class" => "crm-select2")
-    );
-
-    $this->add(
-      'select',
-      'volunteer_default_profiles_group',
-      ts('Default Profile(s) for Group Registration'),
-      $profileList,
-      false, // is required,
-      array("placeholder" => ts("-- No Default profiles --"), "multiple" => "multiple", "class" => "crm-select2")
-    );
-
-    $this->add(
-      'select',
-      'volunteer_default_profiles_both',
-      ts('Default Profile(s) for Both Individual and Group Registration'),
-      $profileList,
-      false, // is required,
-      array("placeholder" => ts("-- No Default profiles --"), "multiple" => "multiple", "class" => "crm-select2")
-    );
+    foreach(CRM_Volunteer_BAO_Project::getProjectProfileAudienceTypes() as $audience) {
+      $this->add(
+        'select',
+        'volunteer_default_profiles_' . $audience['type'],
+        ts($audience['description']),
```

Comment:
This doesn't work. The original strings need to be ts'ed, not the variables that represent them.